packages/ckb/src/collector/offline.ts

```diff
@@ -0,0 +1,102 @@
+import { IndexerCell } from '../types/collector';
+import { Collector } from './index';
+import { isScriptEqual } from '../utils/ckb-tx';
+import { Hex } from '../types';
+export interface CellWithStatus extends IndexerCell {
+  status?: 'live';
```

Comment:
I think the `status` and `outputDataHash` are redundant, and the offline collector does not need them.

Do you agree?